runtime-modules/working-group/src/errors.rs

```diff
@@ -265,6 +265,42 @@ decl_error! {
         /// Invalid OpeningPolicyCommitment parameter:
         /// fill_opening_successful_applicant_application_stake_unstaking_period should be non-zero.
         FillOpeningSuccessfulApplicantApplicationStakeUnstakingPeriodIsZero,
+
```

Comment:
I wonder wouldn't a single general `UnstakingPeriodIsZero` have been sufficient? 
Then you wouldn't have to pass the different variants as arguments in `fn check_unstaking_period()` and `fn check_staking_policy()`

The downside is the caller wouldn't be able to identify in which policy they made a mistake, but it would simplify the code, and reduce chance of returning the incorrect error variant.

Reply:
I thought about it because it would be easier to implement, but I wanted to avoid the downside you mentioned.